src/yb/master/catalog_entity_info.cc

```diff
@@ -413,12 +413,12 @@ bool TableInfo::IsAlterInProgress(uint32_t version) const {
 }
 
 bool TableInfo::HasTablets() const {
-  shared_lock<decltype(lock_)> l(lock_);
+  SharedLock<decltype(lock_)> l(lock_);
   return !tablet_map_.empty();
 }
 
 bool TableInfo::AreAllTabletsDeletedOrHidden() const {
-  shared_lock<decltype(lock_)> l(lock_);
+  SharedLock<decltype(lock_)> l(lock_);
```

Comment:
There have been changes to `catalog_entity_info.cc` on `master` since this PR was originally submitted. I took the liberty of resolving the resulting merge conflicts, adding `SharedLock` to a couple of additional functions.